tools/test/examples/README.md

```diff
@@ -0,0 +1,81 @@
+# Examples testing script
+
+examples.py is script which makes easier to test examples.
+```
+usage: examples.py [-h] [-c CONFIG] [-e EXAMPLE] [-p TEST_PHASE]
+                   {import,clone,deploy,tag,compile,export} ...
+
+positional arguments:
+  {import,clone,deploy,tag,compile,export}
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -c CONFIG
+  -e EXAMPLE, --example EXAMPLE
+                        filter the examples used in the script
```

Comment:
Does this mean 'remove an example' or 'add an example'? Can we have multiple examples?

Reply:
as it says, it is filtering

Reply:
My apologies, I'll clarify my question. Does '-e X' test only example 'X', or does it test all specified examples except 'X'?